docs_src/dependencies/tutorial005_an.py

```diff
@@ -21,6 +21,6 @@ def query_or_cookie_extractor(
 
 @app.get("/items/")
 async def read_query(
-    query_or_default: Annotated[str, Depends(query_or_cookie_extractor)]
+    query_or_default: Annotated[str, Depends(query_or_cookie_extractor)],
```

Comment:
This is a change that was also made in latest Black: https://github.com/psf/black/pull/4164.